app/presenters/two_factor_authentication/set_up_selection_presenter.rb

```diff
@@ -32,6 +46,26 @@ def mfa_added_label
       end
     end
 
+    def visible?
+      if piv_cac_required?
+        type == :piv_cac
+      elsif phishing_resistant_required?
+        phishing_resistant?
+      elsif desktop_only?
```

Comment:
Would this line be considered as a bit of mental gymnastics? I was thinking `!browser.mobile?` could go into something like `is_browser_mobile?` for readability.

Reply:
Yeah, to the readability point, I initially tried `browser.desktop?` to avoid the negation, before realizing that the method doesn't actually exist and fixing in 7a1239d1b7e6b02ad9d855b6965ffcfaca31c9ef.

Personally I think the negation isn't too bad to warrant adding another method here.